README.md

```diff
@@ -7,6 +7,65 @@ This is the Ruby library for Rollbar. It will instrument many kinds of Ruby appl
 
 <!-- Sub:[TOC] -->
```

Comment:
You can/should remove this (line 8) -- this is what generates the table of contents that we show at https://rollbar.com/docs/notifier/rollbar.js/ .